boxyhq/saas-starter-kit
# Stripe Integration

This PR adds Stripe Integration to the saas-starter-kit. Fixes #12 

How to test:
 - Create an account on Stripe.
 - Get the `Secret Key` from [API Keys](https://dashboard.stripe.com/test/apikeys) section.
 - Install Stripe Cli and authenticate
 - Create products and prices using the Stripe dashboard
 - Run `npm run sync-stripe` to ensure db & stripe have the same set of active products and prices.
 - Run `stripe listen --forward-to=localhost:4002/api/webhooks/stripe` in a terminal to forward the webhook events to local apis
 - Run the app with `npm run dev`
 - Test Credit Card number: `4242 4242 4242 4242`
